examples/submeter_hierarchies/analyze.py

```diff
@@ -0,0 +1,78 @@
+import brickschema
+
+
+def get_graph(filename):
+    g = brickschema.Graph()
+    # load local Brick file
+    g.load_file("../../Brick.ttl")
+    g.load_file(filename)
+    g.expand("shacl")
+    return g
+
+
+FILES = [
+    "main-and-submeter.ttl",
+    "multiple-submeters.ttl",
+    "single-meter.ttl",
+]
+
```

Comment:
Can we add another example for accommodating Collections? This will help us to validate how the collections/non-collections would work for `meters`.

Reply:
There's one in the docs -- is that enough?